<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.context.config.ContextNamespaceHandlerTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2021-02-23T08:38:37" hostname="CHENGXINHUA-PC" time="0.116">
  <properties/>
  <testcase name="propertyPlaceholderLocationWithSystemPropertyForMultipleLocations()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.022"/>
  <testcase name="propertyPlaceholderLocationWithSystemPropertyMissing()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.01"/>
  <testcase name="propertyPlaceholderLocation()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.016"/>
  <testcase name="propertyPlaceholderLocationWithSystemPropertyForOneLocation()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.013"/>
  <testcase name="propertyPlaceholderIgnored()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.012"/>
  <testcase name="propertyOverride()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.012"/>
  <testcase name="propertyPlaceholderEnvironmentProperties()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.011"/>
  <testcase name="propertyPlaceholderSystemProperties()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.009"/>
  <testcase name="propertyPlaceholder()" classname="org.springframework.context.config.ContextNamespaceHandlerTests" time="0.01"/>
  <system-out><![CDATA[16:38:37.183 [Test worker] WARN  o.s.c.s.ClassPathXmlApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'foo' defined in class path resource [org/springframework/context/config/contextNamespaceHandlerTests-location-placeholder.xml]: Could not resolve placeholder 'foo' in value "${foo}"; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'foo' in value "${foo}"
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
